doc/Language/modules.pod6

```diff
@@ -524,6 +524,113 @@ The directories in the C<RAKULIB> path will be searched for modules when
 Raku C<need>s, C<use>s or C<require>s them. Directories that start with a
 dot are ignored and symlinks are followed.
 
+=head1 Testing modules and a distribution
+
+It is important in this section to repeat the note at the beginning of this document, namely that
+there is a difference between a B<Raku> C<distribution>, which approximates a I<module> in other languages, 
```

Comment:
A distribution is a collection of modules and other code, packaged in such a way as to be distributed. I wouldn’t draw a comparison between a distribution and a module; one is simply part of the other.